### PR TITLE
[RW-9601][risk=no] Simplify zone logic

### PR DESCRIPTION
**Pull request by evrii**
Added commit that I forgot to merge in https://github.com/all-of-us/workbench/pull/8812 

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/components/runtime-configuration-panel/customize-panel.tsx [hunk 1]
```diff
@@ -253,13 +252,13 @@ export const CustomizePanel = ({
...
                 If you would like to change zones, please make sure that you first delete both your environment and your persistent disk`}
-                  disabled={canChangeZone}
+                  disabled={!gcePersistentDisk}
                 >
                   <div>
                     <Dropdown
                       id='runtime-zone'
                       appendTo='self'
-                      disabled={disableControls || !canChangeZone}
+                      disabled={disableControls || !!gcePersistentDisk}
```
Comment (jmthibault79) [inline ui/src/app/components/runtime-configuration-panel/customize-panel.tsx:261]: sign flipped

Comment (evrii) [reply ui/src/app/components/runtime-configuration-panel/customize-panel.tsx:261]: disabled={disableControls || !canChangeZone}
before my change this was equivalent to:
`disabled={disableControls || !(!currentRuntime || !gcePersistentDisk)}`
now it equals:
disabled={{disableControls || !( ~~!currentRuntime ||~~ !gcePersistentDisk)}} 
which is is equivalent to:
`disabled={{disableControls || !(!gcePersistentDisk)}}`
`disabled={{disableControls || !!gcePersistentDisk}}`

Would it be more readable to keep changeZone? That way I could add a comment explain what is happening.
`changeZone = !gcePersistentDisk`

Comment (evrii) [reply ui/src/app/components/runtime-configuration-panel/customize-panel.tsx]: No worries. I updated it. Please let me know if you think the comment should be tweaked.

**Review by jmthibault79 — APPROVED** [review]
Looks great, thank you!

**Pull request merged**